core/src/test/java/org/mobilitydata/gtfsvalidator/validator/DefaultFieldValidatorTest.java

```diff
@@ -119,6 +119,19 @@ public void validatePhoneNumber_valid() {
         nc ->
             new DefaultFieldValidator(CountryCode.forStringOrUnknown("NL"))
                 .validatePhoneNumber("004980038762246", CELL_CONTEXT, nc));
+
+    assertValid(
+        nc ->
+            new DefaultFieldValidator(CountryCode.forStringOrUnknown("us"))
+                .validatePhoneNumber("813-254-4278", CELL_CONTEXT, nc));
+
+    // The below format fails validation without a known country code, so check that we skip
+    // validation when country
+    // code is unknown
```

Comment:
Please merge the last two lines and put a full stop at the end.

Reply:
Thanks for catching that! Not sure why the code formatter split the lines, but it's fixed now with the period too in https://github.com/MobilityData/gtfs-validator/pull/1062/commits/5223228e99241ccf2e3395c5822fe8abdf0104cd.